iree/turbine/kernel/wave/utils.py

```diff
@@ -493,7 +548,17 @@ def compile_and_invoke(
 
     if run:
         func = mod.lookup_function(func_name)
-        _invoke(ctx.vm_context, device, func, kernel_inputs, kernel_outputs)
+        if inplace:
+            _inplace_invoke(
+                ctx.vm_context,
+                device,
+                func,
+                kernel_inputs,
+                kernel_outputs,
+                kernel_dynamic_dims,
+            )
+        else:
+            _invoke(ctx.vm_context, device, func, kernel_inputs, kernel_outputs)
```

Comment:
Do we need to modify the signature of _invoke to include the kernel dynamic dims since they are no longer included in the kernel inputs?

Reply:
Good catch, yes, I'll update that :)

Reply:
done!